recipes-core/acrn/acrn-hypervisor/efi-loader.patch

```diff
@@ -1,26 +1,29 @@
-From 87407108c90039540526649b9aeeb9fd3628c077 Mon Sep 17 00:00:00 2001
-From: Ross Burton <ross.burton@intel.com>
-Date: Fri, 15 Mar 2019 11:42:16 +0000
-Subject: [PATCH] Update the default EFI loader to the EFI default, not Clear
- Linux specific.
+From b51f37838e6200722f77b9c116092b745ccb7fbb Mon Sep 17 00:00:00 2001
+From: Naveen Saini <naveen.kumar.saini@intel.com>
+Date: Fri, 27 Dec 2019 10:51:38 +0800
+Subject: [PATCH] kconfig: Update the default EFI loader to the EFI default,
+ not Clear Linux specific.
 
 Upstream-Status: Inappropriate
 Signed-off-by: Ross Burton <ross.burton@intel.com>
-
+Signed-off-by: Naveen Saini <naveen.kumar.saini@intel.com>
 ---
- misc/efi-stub/boot.c | 2 +-
+ hypervisor/arch/x86/Kconfig | 2 +-
  1 file changed, 1 insertion(+), 1 deletion(-)
 
-diff --git a/misc/efi-stub/boot.c b/misc/efi-stub/boot.c
-index 0ff725e..0cb5e61 100644
---- a/misc/efi-stub/boot.c
-+++ b/misc/efi-stub/boot.c
-@@ -347,7 +347,7 @@ static inline EFI_STATUS isspace(CHAR8 ch)
-     return ((uint8_t)ch <= ' ');
- }
+diff --git a/hypervisor/arch/x86/Kconfig b/hypervisor/arch/x86/Kconfig
+index 41a78fe7..f0572bb3 100644
+--- a/hypervisor/arch/x86/Kconfig
++++ b/hypervisor/arch/x86/Kconfig
+@@ -318,7 +318,7 @@ config MAX_KATA_VM_NUM
  
--#define DEFAULT_UEFI_OS_LOADER_NAME L"\\EFI\\org.clearlinux\\bootloaderx64.efi"
-+#define DEFAULT_UEFI_OS_LOADER_NAME L"\\EFI\\BOOT\\bootx64.efi"
- /**
-  * efi_main - The entry point for the OS loader image.
-  * @image: firmware-allocated handle that identifies the image
+ config UEFI_OS_LOADER_NAME
+ 	string "UEFI OS loader name"
+-	default "\\EFI\\org.clearlinux\\bootloaderx64.efi"
++	default "\\EFI\\BOOT\\bootx64.efi"
```

Comment:
Back to kconfig again?  We *should* be able to seed this instead of patching, like we used to do back in the ACRN 1.0 days.

Reply:
Hi @rossburton , yes, we re-introduced the Kconfig parameter, I believe this is what you wanted. I guess the proper way to use this though would be to use a specific `.config` file instead of modifying the default value in `Kconfig`.

See also the PR that re-introduced this parameter: https://github.com/projectacrn/acrn-hypervisor/pull/4019